src/main/java/uk/gov/hmcts/divorce/ccd/tab/CaseTypeTab.java

```diff
@@ -91,5 +91,15 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
         configBuilder.tab("documents", "Documents")
             .field(CaseData::getDocumentsGenerated)
             .field(CaseData::getDocumentsUploaded);
+
+        configBuilder.tab("marriageDetails", "Marriage Certificate")
+            .field(CaseData::getMarriageApplicant1Name)
```

Comment:
I wonder if those fields should go in MarriageDetails too?

Reply:
can move those as a separate PR